test/e2e/install.go

```diff
@@ -44,8 +45,26 @@ type installOptions struct {
 	ResticHelperImage      string
 }
 
+func getPlugins(ctx context.Context, veleroCLI, objectStoreProvider, Plugins string) ([]string, error) {
```

Comment:
Let's move this function into the velero_util.go and merge it with `getProviderPlugins`